assets/css/_modal.scss

```diff
@@ -14,3 +14,11 @@
   @include modal-backdrop;
   z-index: 2000;
 }
+
+.c-modal-fullscreen {
+  z-index: 2001;
```

Comment:
**suggestion:** if you want, a quick and easy refactor could be to move these `z-index` values in this file into the `$z-page-layout-context` map in `app.scss`.